tests/lib/helpers.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+import logging
+import decorator
+
+def disable_loganalyzer(func):

```

Comment:
Can this decorator work with whole class/module?
E.g.: each test case in my test class expects specific error messages (error flow test), I will call loganalyzer API directly, not using a fixture as this fixture will fail for each test case